hammerblade/torch/kernel/hb_reduction.hpp

```diff
@@ -69,9 +69,9 @@ inline void binary_reduction_simple(HBTensor<scalar_t> out,
                                     F1 reduce, F2 project) {
   hb_assert_msg(out.numel() == 1, "reduction_simple only handles trivial case");
 
-  __remote scalar_t* data[2];
-  data[0] = (__remote scalar_t*)out.data_ptr();
-  data[1] = (__remote scalar_t*)in.data_ptr();
+  scalar_t* data[2];
```

Comment:
Hmmm we also dont want to revert what Bandhav did. So these need to be __remote still

Reply:
Yes okay reverted that